client/my-sites/sites/index.jsx

```diff
@@ -124,17 +103,4 @@ export const Sites = createReactClass( {
 	},
 } );
 
-const selectSite = ( siteId, rawPath ) => ( dispatch, getState ) => {
-	const path = rawPath === '/sites' ? '/stats/insights' : rawPath;
-	page( addSiteFragment( path, getSiteSlug( getState(), siteId ) ) );
-};
-
-export default connect(
-	state => {
-		return {
-			selectedSite: getSelectedSite( state ),
-			sites: getSites( state ),
-		};
-	},
-	{ selectSite }
-)( Sites );
+export default connect()( Sites );
```

Comment:
If the `connect` call no longer maps anything from Redux store, it can be completely removed.